File: nowcoder/BC49_判断两个数的大小关系/BC49.c
#include<stdio.h>
int main(){
    int a,b;
    while (scanf("%d %d",&a,&b)!=EOF){
        if(a==b){
        printf ("%d=%d\n",a,b); }
       else if (a>b){
           printf("%d>%d\n",a,b);}
            else if (a<b){
           printf("%d<%d\n",a,b);}
            
    }
  
    return 0;
}

File: nowcoder/BC49_判断两个数的大小关系/BC49.go
package main
  
import "fmt"
  
func main() {
    for {
        var a,b int
        _,err := fmt.Scanf("%d %d\n",&a,&b)
        if err != nil {
            break
        }
        if a > b {
            fmt.Printf("%d>%d\n",a,b)
        } else if a == b {
            fmt.Printf("%d=%d\n",a,b)
        } else {
            fmt.Printf("%d<%d\n",a,b)
        }
    }
}
